Tests/AWSLambdaRuntimeTests/LambdaRuntimeTests.swift

```diff
@@ -50,17 +50,24 @@ struct LambdaRuntimeTests {
             }
 
             // wait a small amount to ensure runtime1 task is started
-            try await Task.sleep(for: .seconds(0.5))
```

Comment:
When I run all the tests in parallel this test fails quite regularly as the second task gets run before the first. I don't think you can rely on a sleep to ensure the two tasks are run in order. Maybe you should care which order they run in, just that one throws a `LambdaRuntimeError`. Do the error check before the `taskGroup.next()`
```swift
await #expect(throws: LambdaRuntimeError.self) {
    try await taskGroup.next()
}
```
I'm pretty sure you can get rid of the test that you can run the second runtime after the failure. Is it really valid to have two separate runtimes, run in the the same lambda?